netsuite/10.0/modules/ROOT/pages/netsuite-reference.adoc

```diff
@@ -1035,7 +1035,7 @@ Used to change the password for the account.
 |===
 | Name | Type | Description | Default Value | Required
 | Configuration | String | The name of the configuration to use. | | x
-| New Password a| String |  New password value. |  `#[payload]` |
+| New Password a| String |  New password value. |   |
 | Current Credentials a| String |  current user password. |  | x
 | Target Variable a| String |  The name of a variable to store the operation's output. |  |
 | Target Value a| String |  An expression to evaluate against the operation's output and store the expression outcome in the target variable |  `#[payload]` |
```

Comment:
The description is confusing:

An expression to evaluate against ...output
and
(an expression to) store...?
(an expression) in which to store...?

What are the two actions and the two actors in this description?